website/docs/getting-started/setup/instance-configuration/http-proxy.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 10
+---
+# HTTP/HTTPS Proxy
+
+Appsmith supports running behind a forward proxy. This is usually useful when Appsmith is run on a server that's behind a corporate firewall and needs to access the internet through a proxy.
+
+## How to configure a proxy
+
+As is standard for configuring a proxy for most applications, Appsmith respects any `HTTP_PROXY` and/or `HTTPS_PROXY` env variables that are set. Fox extra compatibility, Appsmith also respects the lower-case versions, `http_proxy` and `https_proxy`.
+
+For example, adding the following to the `stacks/configuration/docker.env`:
```

Comment:
Are these configuration variables also available for Kubernetes - `values.yaml` file?

Reply:
Yes, they're environment variables, they can be set anywhere where environment variables can be set. This is just an example that I put in, happy to rephrase. 🙂